common/src/main/java/com/microsoft/identity/common/DataBagUtil.java

```diff
@@ -20,36 +20,46 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-package com.microsoft.identity.common.java.eststelemetry;
+package com.microsoft.identity.common;
+
+import android.os.Bundle;
 
 import com.microsoft.identity.common.java.interfaces.INameValueStorage;
+import com.microsoft.identity.common.java.util.ported.DataBag;
 
-import java.util.concurrent.ConcurrentHashMap;
+import java.io.Serializable;
+import java.util.Map;
 
 import lombok.NonNull;
 
-public class TelemetryMap implements INameValueStorage<CurrentRequestTelemetry> {
+/**
+ * {@link DataBag} utilities.
```

Comment:
I noticed that DataBag itself doesn't have a meaningful description in teh code comments.  What is a DataBag?  Generally i've heard of Property bags... but a databag is less clear to me.

Reply:
I can change this to PropertyBag.